plugins/q/src/brush/q-brush.js

```diff
@@ -285,7 +285,7 @@ export default function qBrush(brush, opts = {}, layout) {
   if (methods.rangeSelectHyperCubeValues) {
     selections.push({
       method: 'rangeSelectHyperCubeValues',
-      params: [methods.rangeSelectHyperCubeValues.path, methods.rangeSelectHyperCubeValues.ranges, [], true],
+      params: [methods.rangeSelectHyperCubeValues.path, methods.rangeSelectHyperCubeValues.ranges, [], opts.orMode ?? true],
```

Comment:
Update jsdoc with this new property

Reply:
This is not a new property. Previously, the forth parameter is always true, now we allow to set it to false.

Reply:
I don't see where we address this orMode in jsdoc so I can update.

Reply:
For picasso-plugin-q this is new
and should be documented here: https://github.com/qlik-oss/picasso.js/blob/c427e83781fb667c96b356c4d1a6acee494d9fc4/plugins/q/src/brush/q-brush.js#L114-L119

Reply:
Updated!